pkg/operator/controller/ingress/namespace_test.go

```diff
@@ -0,0 +1,58 @@
+package ingress
+
+import (
+	"testing"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestRouterNamespaceChanged(t *testing.T) {
+	testCases := []struct {
+		description string
+		mutate      func(*corev1.Namespace)
+		expect      bool
+	}{
+		{
+			description: "if nothing changes",
+			mutate:      func(_ *corev1.Namespace) {},
+			expect:      false,
+		},
+		{
+			description: "if namespace network-policy annotation changes",
```

Comment:
```suggestion
			description: "if namespace network-policy label changes",
```
...but it's not worth holding up the PR for this change, and I just realized the same error is in the main branch.

Reply:
I noticed that as well, but decided it wasn't worth the hastle.